instance/models/database_server.py

```diff
@@ -120,6 +121,10 @@ class DatabaseServer(ValidateModelMixin, TimeStampedModel):
     class Meta:
         abstract = True
 
+    # Human readable identifier for database instances
+    name = models.CharField(max_length=128, unique=True)
+    description = models.CharField(max_length=128, blank=True)
```

Comment:
I think it would make sense to somehow unify the database server models and the load balancing server model, since they are almost the same.  They also use custom managers that are very similar.